.changeset/config.js

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
We could probably clean up the comments in this file quite a bit.

Reply:
👍